lib/fog/vsphere/requests/compute/list_clusters.rb

```diff
@@ -41,7 +41,7 @@ def cluster_attributes cluster, datacenter_name
 
         def cluster_path(cluster, datacenter_name)
           datacenter = find_raw_datacenter(datacenter_name)
-          cluster.pretty_path.gsub(/(#{datacenter.name}|#{datacenter.hostFolder.name})\//,'')
+          cluster.pretty_path.gsub(/(#{datacenter_name}|#{datacenter.hostFolder.name})\//,'')
         end
       end
```

Comment:
I was maybe not so understandable. Let me be more clear. This is full path to one of our clusters: <code>vCenter/Germany[folder]/GIO[datacenter]/host[vmware-obj]/GIO[folder]/MGMT[cluster]</code>.
The method <code>cluster_path</code> should return relative path to a cluster (<code>GIO[folder]/MGMT</code> - relative to datacenter) as it is awaited in <code>create_vm.rb</code> when discovering <code>resource_pool</code>. While "cluster.pretty_path" returns full path to cluster beginning with first folder under root, e.g. <code>Germany[folder]/GIO[datacenter]/host[vmware-obj]/GIO[folder]/MGMT[cluster]</code>, full path to datacenter needs to be subtracted. But <code>datacenter.name</code> returns only simple DC name (<code>GIO</code> in this case) while <code>datacenter_name</code> contains path to datacenter (<code>Germany/GIO[datacenter]</code>).